agent/models/agent_message.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package models contains client domain models and helpers.
+package models
+
+import (
+	"google.golang.org/grpc/status"
+
+	"github.com/percona/pmm/api/agentpb"
+)
+
+type AgentMessageType uint8
+
+const (
+	AgentMessageUnknown AgentMessageType = iota
+	AgentMessagePing
+	AgentMessageStateChanged
+	AgentMessageQanCollect
+	AgentMessageActionResult
+	AgentMessageJobResult
+	AgentMessageJobProgress
+	AgentMessagePong
+	AgentMessageSetState
+	AgentMessageStartAction
+	AgentMessageStopAction
+	AgentMessageCheckConnection
+	AgentMessageStartJob
+	AgentMessageStopJob
+	AgentMessageJobStatus
+	AgentMessageGetVersions
+	AgentMessagePbmSwitchPitr
+	AgentMessageAgentLogs
```

Comment:
it seems like it adds extra manual work when we will add a new message type. can we use a `switch` similar to [this one](https://github.com/percona/pmm/pull/1970/files#diff-70d8d230a84815566d99bfdc9a0010ea4739207d137b2cf821efa8aed76cba7eR240)?